### PR TITLE
An info page about a plugin for integrating Simple Analytics with WP

### PR DESCRIPTION
**Pull request by ChrisPlaneta**
This is about the WP Full Picture plugin. I added the info page to the main menu. I don't know if you want to keep it like that or add an info in the existing page about WordPress plugins (like you did with that other WP plugin).

### CODE REVIEW

File: _docs/31_integrations/06_wordpress_wpfullpicture.md [hunk 1]
```diff
@@ -0,0 +1,42 @@
...
+
+If you need more features than the official Simple Analytics plugin, you can try the free version of [WP Full Picture](https://wordpress.org/plugins/full-picture-analytics-cookie-notice/).
+
+WP Full Picture lets you:
+* Load Simple Analytics only in specific countries. Simply enable geolocation module and choose countries that you want to use SA in
+* Do not track page views until pages are actually viewed (e.g. when they are opened in tabs). This works out-of-the-box with no extra setup.
+* Turn off tracking for specific user roles and users who are not logged in ([more information](https://wpfullpicture.com/support/documentation/how-to-exclude-users-from-tracking/))
+* See traffic from some Android applications ([requires extra setup](https://wpfullpicture.com/support/documentation/how-to-get-better-traffic-sources-information/))
+* Use custom domain for bypassing ad-blockers (after setting it up as [described here](https://docs.simpleanalytics.com/bypass-ad-blockers))
+* Join traffic data from multiple websites under one domain without the need to [modfy the script](https://docs.simpleanalytics.com/overwrite-domain-name)
```
Comment (adriaandotcom) [inline _docs/31_integrations/06_wordpress_wpfullpicture.md]: Little typo :)

**Review by adriaandotcom — CHANGES_REQUESTED** [review]

**Comment by adriaandotcom** [comment]
Looks great! Thanks for your time, @ChrisPlaneta. Will publish after that little typo is fixed.

**Comment by ChrisPlaneta** [comment]
Hi Adriaan

I will need your help here. I normally never use GitHub for my projects and
I actually have no idea what typo you are talking about :/ I read
documentation from GitHub and I am more lost then before. See the
attachment. This is what I see and I have no clue where this typo is
marked. Can you point me in the right direction here?

Thanks and have a nice day
Krzysztof

czw., 11 maj 2023 o 13:13 Adriaan ***@***.***> napisał(a):

**Comment by ChrisPlaneta** [comment]
Oh. I think I have got it. However, I'm not sure if you can see the change. Let me know.

**Comment by adriaandotcom** [comment]
Thanks Chris, will merge it. No worries about the typo, will fix if it's still there.

**Pull request merged**